src/main/java/betterquesting/commands/admin/QuestCommandDefaults.java

```diff
@@ -143,7 +144,7 @@ private static void sendChatMessage(
 
     public static void save(@Nullable ICommandSender sender, @Nullable String databaseName, File dataDir) {
         BiFunction<String, String, String> buildFileName =
-                (name, id) -> name.replaceAll("[^a-zA-Z0-9]", "") + "-" + id;
+                (name, id) -> EnumChatFormatting.getTextWithoutFormattingCodes(name).replaceAll("[^a-zA-Z0-9]", "") + "-" + id;
```

Comment:
This function is client-only and would cause a crash on dedicated servers

Reply:
Good catch, thanks!

---

One thing that I noticed while attempting to fix this: if a string literal contains a literal `§`, it appears to be getting turned into `ﾂｧ` at runtime (not cosmetic; this broke my regex). I assume that this is due to either Jabel (if BQ currently has it enabled) or something else in RFG, because I have another mod which gets built by a pre-RFG buildscript, and it does not seem to have this problem. `\u00a7` works just fine, though.

Assuming I am not missing something / have some messed up build settings somehow, we may want to at least put up a warning about this behaviour, since it's tricky to find without a debugger, and IntelliJ wants to correct `\u00a7` to `§` in string literals.

Repro instructions:
  1. Add this somewhere that gets executed:
      ```java
        String test = "§";
        String test2 = "\u00a7";
        bool test3 = test2.contains(test);
      ```
  2. Put breakpoint just after that block; run client with debugger attached; inspect the test values in debugger

I have confirmed this issue with the following setups:
  1. `Run Client` (Java 8) in BetterQuesting, as of the time of this comment
  2. `Run Client (Java 17)` in BetterQuesting, as of the time of this comment
  3. GTNH 2.2.9 (Java 19)

This issue occurred before and after the buildscript update included in this PR.

Reply:
Actually, I need to revert the buildscript update. It looks like it breaks BQ dev client testing. I *think* (not 100% sure) the UUID returned by these two methods no longer agrees, which breaks BQ's quest completion checking:
```
QuestingAPI.getQuestingUUID(Minecraft.getMinecraft().thePlayer)
QuestingAPI.getQuestingUUID(EntityPlayerMP from a net message)
```

Reply:
I'm not sure but does this work?
https://github.com/GTNewHorizons/ModularUI/blob/master/addon.gradle

Reply:
Looks like that fixed it, thanks!

Hmm, should we make this the default? Not sure if there are any cases where we would not want this to be enabled